x/ibc/07-tendermint/client_state.go

```diff
@@ -0,0 +1,279 @@
+package tendermint
+
+import (
+	"errors"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	clientexported "github.com/cosmos/cosmos-sdk/x/ibc/02-client/exported"
+	clienttypes "github.com/cosmos/cosmos-sdk/x/ibc/02-client/types"
+	connectionexported "github.com/cosmos/cosmos-sdk/x/ibc/03-connection/exported"
+	channelexported "github.com/cosmos/cosmos-sdk/x/ibc/04-channel/exported"
+	commitment "github.com/cosmos/cosmos-sdk/x/ibc/23-commitment"
+	ibctypes "github.com/cosmos/cosmos-sdk/x/ibc/types"
```

Comment:
Just a nit, but these paths are not really standardised anymore, not sure if they should live in `x/ibc/types`.

Reply:
I'm having a hard time figuring up dependencies and how the new client types affect standardization. I propose to revisit this on the ICS06 implementation (ref #5382)

Reply:
Fair enough!